index.html

```diff
@@ -2,9 +2,135 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <meta name="description" content="Lesson 3 homework">
+    <meta name="keywords" content="HTML">
+    <meta name="author" content="Viyaleta Slemeneva">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lesson 3 homework</title>
 </head>
 <body>
+<img src="https://www.history.com/.image/ar_1:1%2Cc_fill%2Ccs_srgb%2Cfl_progressive%2Cq_auto:good%2Cw_1200/MTk1NzY0MDg3NTEzMTYzNDEy/gettyimages-142198198.jpg"
+     width="50%" alt="Eiffel Towe">
+<ol>
+    <li>Знакомство с Web</li>
+    <ul>
+        <li>Краткая история Web и что такое интернет</li>
+        <li>Как устроен браузер и как он получает страницы по сети</li>
+        <li>Что такое Git?</li>
+        <li>Создание первого репозитория</li>
+        <li>О файле .gitignore</li>
+    </ul>
 
+    <li>HTML-верстка- первые шаги</li>
+    <ul>
+        <li>Настройка рабочей среды</li>
+        <li>Краткая история HTML</li>
+        <li>Какие бывают теги и что такое атрибуты</li>
+        <li>Теги для работы с текстом</li>
+        <li>Создание собственной страницы</li>
+    </ul>
+
+    <li>Погружение в HTML-разметку</li>
+    <ul>
+        <li>Мета-теги</li>
+        <li>Мнемоники</li>
+        <li>Тег для работы с изображениями</li>
+        <li>Теги для работы со списками</li>
+        <li>Теги для работы с таблицами</li>
+        <li>Теги для работы с формами</li>
+        <li>Div-блоки</li>
+    </ul>
+</ol>
+<table border="1" width="100%">
```

Comment:
Good homework, but I think that here(44 line) the attributes do not need to be specified in integers, if you want to add something, then you need to use pixels(px) after the integers